bus-mapping/src/evm/opcodes/ids.rs

```diff
@@ -500,7 +500,7 @@ impl OpcodeId {
             OpcodeId::SMOD => GasCost::FAST,
             OpcodeId::ADDMOD => GasCost::MID,
             OpcodeId::MULMOD => GasCost::MID,
-            OpcodeId::EXP => GasCost::ZERO,
+            OpcodeId::EXP => GasCost::SLOW,
```

Comment:
@icemelon The constant gas cost for `EXP` opcode is 10
Reference: [1](https://evm.codes) and [2](https://github.com/ethereum/solidity/blob/c61610302aa2bfa029715b534719d25fe3949059/libevmasm/GasMeter.h#L49)